src/pudl/transform/eia861.py

```diff
@@ -2466,7 +2466,8 @@ def transform(raw_dfs, eia861_tables=PUDL_TABLES["eia861"]):
     # for each of the tables, run the respective transform funtction
     for table in eia861_tables:
         if table not in tfr_funcs.keys():
-            raise ValueError(f"Unrecognized EIA 861 table: {table}")
+            # Skip tables with no transform function implemented
```

Comment:
Were there some tables that show up in the `get_resource_ids()` output that don't have transform functions?

Reply:
This seems like a problem. I think it's resulting in a bunch of code not getting run -- the test coverage dropped by 3% and it seems to be due to EIA-861 and FERC-714 tables that aren't being transformed.

I think the problem is that there's not a one-to-one correspondence between the tables and the functions. Some functions take one input dataframe and output 2-3 dataframes which have been reshaped. So in this case the "tables" that can be requested were referring more to the source tables, not the final database output tables.

* One way to deal with this would be to infer the source tables that need to be processed based on the output tables that have been requested in the settings validation -- so if someone asks for `demand_side_management_misc_eia861` we know that we need to process the `demand_side_management_eia861` input table (and use its associated transform function).
* Another option would be to just process all the tables all the time -- which is what happens now anyway with the ad-hoc ETL process that's running for both the EIA-861 and FERC-714 data sources.
* The real long-term answer is probably to refactor how we deal with these data dependencies, which is the kind of thing Dagster is trying to manage.